src/changelog/2024/09/notifications-bulk-actions.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Managing Notifications
+description: "Interact with notifications"
+date: 2024-09-25 10:00:00.0
+
+authors: ["serban-costin"]
+tags:
+    - changelog
+---
+
+Starting with the v2.10.0 of FlowFuse, users have the ability to manage notifications in bulk, allowing users to select/deselect, mark as read, or mark as unread multiple notifications simultaneously.
```

Comment:
V2.10.0 is not due for a few weeks. Should we be stating "starting with v2.10.0"?

Reply:
It would fall under the v2.10.0 release, and other changelog entries specify the release. I can remove it if needed.

Reply:
> Lastly, this article is placed in the 09 folder and is dated as 09 - this should be updated to reflect time of release

Moved under 10 folder

Reply:
Yeah, we can leave it out for now, the changelog reflects FlowFuse Cloud too, so avoiding version numbers is okay. In our release blog/post that'll cover what features go out into the self-hosted bundles.

Reply:
Thanks Joe. Suggestion offered here:  https://github.com/FlowFuse/website/pull/2647#discussion_r1790321479